MAINTAINERS

```diff
@@ -1,3 +1,15 @@
 Brian Exelbierd <bexelbie@redhat.com> @bexelbie
 Navid Shaikh <nshaikh@redhat.com> @navidshaikh
 Lalatendu Mohanty <lmohanty@redhat.com> @LalatenduMohanty
+
+## How to release the vagrant-service-manager plugin
+
+1. Put the gemfile in pkg/ with `rake build`
```

Comment:
I'd like it to land here, pending further reorg of the docs.  We need this somewhere.